portfolio/src/test/java/com/google/sps/CollectTasksFromJsonArrayTest.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collection;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CollectTasksFromJsonArrayTest {
```

Comment:
Tests in this file can go into the ServletHelperTest

Reply:
Done.